content/docs/cucumber/state.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: State
 subtitle: Sharing state, isolated state, dependency injection
+polyglot:
+- java
+- javascript
+- ruby
+- kotlin
 ---
```

Comment:
Oh, ok, so you made this "polyglot", and that appears to make it have those tabs and have language-specific verbiage on the page appear only for the appropriate language?

Reply:
Exactly!

Reply:
Yes, exactly. Before the page has information on different languages which was confusing. This PR solves that.